### DISCUSSION TITLE
Name not added to contributor list after contribution.

### DISCUSSION THREAD

**Discussion by KevinRodrigues05** [community/community#50883] [Repositories]
### Select Topic Area

Question

### Body

I don't see my name/account in the contributors list after the pull request is merged with a repository. How to resolve this issue?

**Comment by Anmol-Baranwal** [comment]
If your pull request has been successfully merged, but your name or account is not appearing in the contributors list, there are a few possible reasons why this could be happening:

1. Your email address is not associated with your GitHub account: GitHub uses your email address to match your contributions with your GitHub account. If your email address is not associated with your GitHub account, then your contributions may not be recognized. To fix this issue, you can add your email address to your GitHub account by going to your account settings and then selecting "Emails" from the left-hand menu.

2. Your contributions may not meet the criteria for recognition: Some repositories have specific criteria for recognizing contributors, such as requiring a certain number of contributions or specific types of contributions. If your contributions do not meet these criteria, then your name may not appear in the contributors list. To check the criteria for recognition, you can review the repository's documentation or ask the repository owner or maintainers.

3. There may be a delay in updating the contributors list: Sometimes it can take a few minutes or even a few hours for the contributors list to update after a pull request is merged. If you just merged your pull request, it's possible that your name will appear in the contributors list after a little while.

If you have checked these possibilities and your name still does not appear in the contributors list, you can reach out to the repository owner or maintainers for further assistance.

**Reply by KevinRodrigues05** [reply]
Thanks @Anmol-Baranwal for replying. Looks like I have already taken care of all of the points you've suggested also its been over a week now that name is still not added to the repo contributors. I have also contacted the repo owner but he doesn't seem to have an idea about how it works. Here's the link to my contribution https://github.com/danielhrisca/asammdf/pull/824 and the link where my name is included in the release notes https://github.com/danielhrisca/asammdf/releases/tag/7.3.3 
I do not have an idea what to do next. Please could you help me further.

**Reply by KevinRodrigues05** [reply]
I noticed that there was a different email id associated with repo. Now that I have reset it to the right one. Will my account be mentioned in the contributors list?

**Reply by Anmol-Baranwal** [reply]
I apologize for not being able to respond earlier, as I was preoccupied with my exams. Thank you for your patience and understanding.

Yes, resetting the email associated with your GitHub account to the correct one should help ensure that your contributions are recognized and your name is included in the contributors list. GitHub uses your email address to match your contributions with your GitHub account, so it's important that the correct email is associated with your account. 

- If the email address associated with your GitHub account now matches the email address associated with your contribution in the repository, there's a good chance that your account will be correctly attributed as a contributor in the repository.

- After you have updated your email, you can try reaching out to the repository owner or maintainers again to see if they can help update the contributors list or if they have any further suggestions. (This might not be the case, if previous one works)

Additionally, you may want to wait a little while to allow for any changes to propagate through GitHub's system before checking the contributors list again. I hope this helps!

**Reply by KevinRodrigues05** [reply]
Thank you @Anmol-Baranwal for the great help. The issue is resolved now. 😀

**Reply by Anmol-Baranwal** [reply]
@KevinRodrigues05 
I just wanted to check in and see if you have any remaining questions or concerns about my answer. If not, would you please mark my response as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.
